conf/application.conf

```diff
@@ -50,4 +50,4 @@ get-address-io-api {
     key = "q0M6MVJQm0CslqDvMcKUDQ6396"
 }
 
-include file("/etc/gu/subscriptions-frontend.conf")
+include file("/etc/gu/subscriptions-frontend.private.conf")
```

Comment:
You should also update the aws command in README.md to reflect the download of the new file location.